gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2084,6 +2088,14 @@ func Preprocess(store Store, ctx BlockNode, n Node) Node {
 				} else if len(n.Values) != 0 && numNames != len(n.Values) {
 					panic("should not happen")
```

Comment:
can you also modify this message as what you did above?

Reply:
and add a test like this:
```go
package main

var a, b, c = 1, 2

func main() {
}
``` 
correspondingly.

Reply:
should be added here: 89d1130af546002437b6ce2daa47553573bd30c8

Reply:
@ltzmaxwell  is it good for you ?